app/assets/javascripts/src/Metamaps.GlobalUI.js

```diff
@@ -51,31 +51,26 @@ $(document).ready(function () {
           Metamaps[prop].hasOwnProperty('init') &&
           typeof (Metamaps[prop].init) == 'function'
       ) {
-          Metamaps[prop].init();
+          Metamaps[prop].init()
       }
   }
   // load whichever page you are on
   if (Metamaps.currentSection === "explore") {
-      var capitalize = Metamaps.currentPage.charAt(0).toUpperCase() + Metamaps.currentPage.slice(1);
+      var capitalize = Metamaps.currentPage.charAt(0).toUpperCase() + Metamaps.currentPage.slice(1)
 
-      Metamaps.Views.exploreMaps.setCollection( Metamaps.Maps[capitalize] );
+      Metamaps.Views.exploreMaps.setCollection( Metamaps.Maps[capitalize] )
       if (Metamaps.currentPage === "mapper") {
-          Metamaps.Views.exploreMaps.fetchUserThenRender();
-          Metamaps.Header.fetchUserThenChangeSection(!!Metamaps.Active.Mapper, Metamaps.Maps.Mapper.mapperId)
```

Comment:
The elimination of these two separate calls for the user cut down the load time for loading up a 'mappers maps' page